lib/locreator.js

```diff
@@ -12,6 +13,15 @@ var Locreator = function (nemo) {
   this.drivex = Drivex(nemo.driver, nemo.wd);
   this.locatex = Locatex(nemo);
   this.normify = this.normalize = function (locator) {
+    if(locator.constructor !== String) {
+      if(!locator.locator || !locator.locator.trim()) {
+        throw new Error('NO or EMPTY locator found, please fix it');
+      }
+      if(!locator.type || !locator.type.trim() || !nemo.wd.By[locator.type]) {
```

Comment:
I think I'd rather be in favor of throwing an error in this scenario as well. If users are using an Object to specify a locator, there are two required pieces, locator and type. If those are missing, or in the case of type it is missing or invalid, we should throw and error. The current approach masks a user error and makes debugging more difficult.